hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/ProtobufLogReader.java

```diff
@@ -424,7 +424,7 @@ protected boolean readNext(Entry entry) throws IOException {
       } else {
         // Else restore our position to original location in hope that next time through we will
         // read successfully.
-        LOG.warn("Encountered a malformed edit, seeking back to last good position in file, "
+        LOG.debug("Encountered a malformed edit, seeking back to last good position in file, "
```

Comment:
@busbey This is the frequent message. The particular EOFexception it prints originates from https://github.com/apache/hbase/blob/5195981d995482fa1763d6426cf449f2589a2107/hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/ProtobufLogReader.java#L368

Reply:
Yeah I agree that's not useful.